app/controllers/api/accounts_controller.rb

```diff
@@ -1,17 +1,22 @@
 # encoding: utf-8
 
+require 'abstract_resource'
+
 class Api::AccountsController < Api::BaseController
+  include AbstractResource
+
   api_versions :v1
 
   represent_with Api::AccountRepresenter
 
   # filter_resources_by :user_id
 
+  # Unless there's already an opener, set it to the authenticated user
   def create_resource
-    return super unless user
-
-    user.create_individual_account
-    user.individual_account
+    super.tap do |acct|
+      acct.opener_id ||= current_user.id
```

Comment:
Should a user be able to create a GroupAccount/Account here via `kind` param? Or just an IndividualAccount?

Reply:
My understanding is that any user should be able to create a GroupAccount/StationAccount without restriction, since there's additional logic in place for administrators to approve those other account types. The policy that would handle those other account types is the existing account policy, which just checks for the presence of a token.

Reply:
I don't _think_ GroupAccounts need admin approval?  But yeah, StationAccounts definitely do.  The only restriction is probably that the creating user has to have a confirmed email.  (But ID shouldn't be giving out tokens to unconfirmed users anyways).